folium/plugins/image_overlay.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+"""
+Image Overlay
+-------------
+
+Used to load and display a single image over specific bounds of
+the map, implements ILayer interface.
+
+"""
+import json
+from jinja2 import Template
+
+from folium.utilities import image_to_url
+from folium.map import Layer
+
+def mercator_transform(data, lat_bounds, origin='upper', height_out=None):
```

Comment:
Web mercator? (Worth adding the reference to https://en.wikipedia.org/wiki/Web_Mercator in the docstring)